### ISSUE TITLE
[trellis2] ✗ Trellis2ImageToShape.generate failed after 0.01s: Autotuner.__init__() takes from 7 to 13 positional

### ISSUE THREAD

**Issue by Fagnerfriedhofer** [PozzettiAndrea/ComfyUI-TRELLIS2#62]
<img width="769" height="715" alt="Image" src="https://github.com/user-attachments/assets/96103378-b46c-4c47-bad9-f801f3e55809" />

some one receive  this error?

> [START] Security scan
> [ComfyUI-Manager] Using `uv` as Python module for pip operations.
> [DONE] Security scan
> ## ComfyUI-Manager: installing dependencies done.
> ** ComfyUI startup time: 2026-01-19 15:20:15.228
> ** Platform: Linux
> ** Python version: 3.12.12 (main, Dec 24 2025, 11:39:05) [GCC 15.2.1 20251112]
> ** Python executable: /home/fagner/ComfyUI/venv/bin/python
> ** ComfyUI Path: /home/fagner/ComfyUI
> ** ComfyUI Base Folder Path: /home/fagner/ComfyUI
> ** User directory: /home/fagner/ComfyUI/user
> ** ComfyUI-Manager config path: /home/fagner/ComfyUI/user/__manager/config.ini
> ** Log path: /home/fagner/ComfyUI/user/comfyui.log
> [GeometryPack] Parsed GeomPackRemesh: 6 backends
> [GeometryPack] Parsed GeomPackFillHoles: 2 backends
> [GeometryPack] Generated backend_mappings.json
> [GeometryPack] All assets already exist in /home/fagner/ComfyUI/input/3d
> 
> Prestartup times for custom nodes:
>    0.0 seconds: /home/fagner/ComfyUI/custom_nodes/ComfyUI-TRELLIS2
>    0.0 seconds: /home/fagner/ComfyUI/custom_nodes/comfyui-geometrypack
>    0.3 seconds: /home/fagner/ComfyUI/custom_nodes/ComfyUI-Manager
> 

> Checkpoint files will always be loaded safely.
> Total VRAM 11162 MB, total RAM 32012 MB
> pytorch version: 2.8.0+cu126
> Set vram state to: NORMAL_VRAM
> Device: cuda:0 NVIDIA GeForce GTX 1080 Ti : cudaMallocAsync
> Using async weight offloading with 2 streams
> Enabled pinned memory 30411.0
> WARNING: You need pytorch with cu130 or higher to use optimized CUDA operations.
> Found comfy_kitchen backend cuda: {'available': True, 'disabled': True, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8']}
> Found comfy_kitchen backend triton: {'available': True, 'disabled': True, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8']}
> Found comfy_kitchen backend eager: {'available': True, 'disabled': False, 'unavailable_reason': None, 'capabilities': ['apply_rope', 'apply_rope1', 'dequantize_nvfp4', 'dequantize_per_tensor_fp8', 'quantize_nvfp4', 'quantize_per_tensor_fp8', 'scaled_mm_nvfp4']}
> Using pytorch attention
> Python version: 3.12.12 (main, Dec 24 2025, 11:39:05) [GCC 15.2.1 20251112]
> ComfyUI version: 0.9.2
> ComfyUI frontend version: 1.36.14
> [Prompt Server] web root: /home/fagner/ComfyUI/venv/lib/python3.12/site-packages/comfyui_frontend_package/static
> ### Loading: ComfyUI-Manager (V3.39.2)
> [ComfyUI-Manager] network_mode: public
> [ComfyUI-Manager] ComfyUI per-queue preview override detected (PR #11261). Manager's preview method feature is disabled. Use ComfyUI's --preview-method CLI option or 'Settings > Execution > Live preview method'.
> ### ComfyUI Revision: 4551 [8f40b43e] *DETACHED | Released on '2026-01-15'
> [ComfyUI-TRELLIS2] Initializing custom node...
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [trellis2] Auto-discovered v2 config: trellis2
> [ComfyUI-TRELLIS2] [OK] Node classes imported successfully
> [ComfyUI-TRELLIS2] [OK] Loaded successfully!
> [GeomPack] CGAL Python package found - CGAL Isotropic Remesh node available
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
> [GeomPack] Custom server routes registered
> 
> Import times for custom nodes:
>    0.0 seconds: /home/fagner/ComfyUI/custom_nodes/websocket_image_save.py
>    0.0 seconds: /home/fagner/ComfyUI/custom_nodes/ComfyUI-Manager
>    0.0 seconds: /home/fagner/ComfyUI/custom_nodes/ComfyUI-TRELLIS2
>    0.9 seconds: /home/fagner/ComfyUI/custom_nodes/comfyui-geometrypack
> 
> Context impl SQLiteImpl.
> Will assume non-transactional DDL.
> Assets scan(roots=['models']) completed in 0.016s (created=0, skipped_existing=12, total_seen=12)
> Starting server
> 
> To see the GUI go to: http://127.0.0.1:8188
> FETCH ComfyRegistry Data: 5/120
> FETCH ComfyRegistry Data: 10/120
> FETCH ComfyRegistry Data: 15/120
> FETCH ComfyRegistry Data: 20/120
> FETCH ComfyRegistry Data: 25/120
> FETCH ComfyRegistry Data: 30/120
> FETCH ComfyRegistry Data: 35/120
> FETCH ComfyRegistry Data: 40/120
> FETCH ComfyRegistry Data: 45/120
> FETCH ComfyRegistry Data: 50/120
> FETCH ComfyRegistry Data: 55/120
> FETCH ComfyRegistry Data: 60/120
> FETCH ComfyRegistry Data: 65/120
> FETCH ComfyRegistry Data: 70/120
> FETCH ComfyRegistry Data: 75/120
> FETCH ComfyRegistry Data: 80/120
> FETCH ComfyRegistry Data: 85/120
> FETCH ComfyRegistry Data: 90/120
> FETCH ComfyRegistry Data: 95/120
> FETCH ComfyRegistry Data: 100/120
> FETCH ComfyRegistry Data: 105/120
> FETCH ComfyRegistry Data: 110/120
> FETCH ComfyRegistry Data: 115/120
> FETCH ComfyRegistry Data: 120/120
> FETCH ComfyRegistry Data [DONE]
> [ComfyUI-Manager] default cache updated: https://api.comfy.org/nodes
> FETCH DATA from: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json [DONE]
> [ComfyUI-Manager] All startup tasks have been completed.
> [DEPRECATION WARNING] Detected import of deprecated legacy API: /scripts/ui.js. This is likely caused by a custom node extension using outdated APIs. Please update your extensions or contact the extension author for an updated version.
> [DEPRECATION WARNING] Detected import of deprecated legacy API: /extensions/core/groupNode.js. This is likely caused by a custom node extension using outdated APIs. Please update your extensions or contact the extension author for an updated version.
> [DEPRECATION WARNING] Detected import of deprecated legacy API: /scripts/ui/components/buttonGroup.js. This is likely caused by a custom node extension using outdated APIs. Please update your extensions or contact the extension author for an updated version.
> [DEPRECATION WARNING] Detected import of deprecated legacy API: /scripts/ui/components/button.js. This is likely caused by a custom node extension using outdated APIs. Please update your extensions or contact the extension author for an updated version.
> got prompt
> Failed to validate prompt for output 69:
> * Trellis2ImageToShape 93:
>   - Value 49152 bigger than max of 50: shape_sampling_steps
> Output will be ignored
> Failed to validate prompt for output 94:
> Output will be ignored
> Prompt executed in 0.08 seconds
> got prompt
> Failed to validate prompt for output 69:
> * Trellis2ImageToShape 93:
>   - Value 49152 bigger than max of 50: shape_sampling_steps
> Output will be ignored
> Failed to validate prompt for output 94:
> Output will be ignored
> Prompt executed in 0.00 seconds
> got prompt
> [trellis2] Creating PersistentVenvWorker (python=/home/fagner/ComfyUI/custom_nodes/ComfyUI-TRELLIS2/_env_trellis2/bin/python)
> [trellis2] → Trellis2GetConditioning.get_conditioning(model_config=Trellis2ModelConfig, image=Tensor([1, 1024, 1024, 3], torch.float32, cpu, 12.0MB), mask=Tensor([1, 1024, 1024], torch.float32, cpu, 4.0MB), include_1024=bool, background_color=str)
>   [TRELLIS2] Running conditioning...
>   [ATTENTION] Backend set to: flash_attn
>   [SPARSE] Attention backend set to: flash_attn
>   [TRELLIS2] Attention backend set to: flash_attn
>   [TRELLIS2] LazyModelManager initialized (resolution=1024_cascade, vram_mode=keep_loaded: keep all models in VRAM)
>   [TRELLIS2] Loading DinoV3 feature extractor...
>   [ComfyUI-TRELLIS2] Remapping facebook/dinov3-vitl16-pretrain-lvd1689m -> PIA-SPACE-LAB/dinov3-vitl-pretrain-lvd1689m
>   [TRELLIS2] Checking for local DINOv3 safetensors in: /home/fagner/ComfyUI/models/dinov3
>   [ComfyUI-TRELLIS2] Downloading DINOv3 model: PIA-SPACE-LAB/dinov3-vitl-pretrain-lvd1689m...
>   [ComfyUI-TRELLIS2] TIP: For cleaner storage, download model.safetensors directly to /home/fagner/ComfyUI/models/dinov3/
>   [ComfyUI-TRELLIS2] DINOv3 model loaded successfully
>   [TRELLIS2] DinoV3 loaded successfully
>   [TRELLIS2] DinoV3 offloaded
>   [TRELLIS2] Conditioning extracted

**[trellis2] ← Trellis2GetConditioning.get_conditioning returned tuple(2 items) [23.87s]
[trellis2] → Trellis2ImageToShape.generate(model_config=Trellis2ModelConfig, conditioning=dict, seed=int, ss_guidance_strength=float, ss_sampling_steps=int, shape_guidance_strength=float, shape_sampling_steps=int, max_tokens=int)
  [TRELLIS2] Running shape generation (seed=42)...
[trellis2] ✗ Trellis2ImageToShape.generate failed after 0.76s: Autotuner.__init__() takes from 7 to 13 positional arguments but 14 were given**

**Comment by PozzettiAndrea** [comment]
@Fagnerfriedhofer Closing this issue as stale. Please do let me know if you still have issues, always here to help :)
